multicluster/build/yamls/antrea-multicluster-leader-global.yml

```diff
@@ -1125,6 +1125,17 @@ spec:
                                   traffic must match. If not specified, this field
                                   defaults to TCP.
                                 type: string
+                              sourceEndPort:
+                                description: SourceEndPort defines the end of the
+                                  source port range, inclusive.
```

Comment:
+ It can only be specified when a `sourcePort` is specified.

Reply:
Done